package.json

```diff
@@ -6,8 +6,10 @@
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
     "web-vitals": "^1.1.2"
   },
```

Comment:
- [ ] react-redux package is missing in the dependencies, please install it. This is part of requirement number two which I'll quote below.

> - Add React Redux (`npm install react-redux`).